torch/csrc/jit/passes/quantization/qat_combine_conv_bn.cpp

```diff
@@ -0,0 +1,90 @@
+#include <torch/csrc/jit/passes/quantization/qat_combine_conv_bn.h>
+
+#include <torch/csrc/jit/ir/subgraph_matcher.h>
+#include <torch/csrc/jit/jit_log.h>
+#include <torch/csrc/jit/passes/graph_rewrite_helper.h>
+#include <torch/csrc/jit/passes/subgraph_rewrite.h>
+#include <torch/csrc/jit/passes/quantization/helper.h>
+
+namespace torch {
+namespace jit {
+
+namespace {
+using graph_rewrite_helper::PatternInfo;
+
+class QATCombineConvBatchNorm2dHelper {
+ public:
+  void transform(Module& module);
+};
+
+std::pair<std::string, std::string> getBeforeAndAfterPatterns() {
```

Comment:
would be better to put the patterns and filters together

Reply:
[optional] nit: rename this function to `getQATConvBnPattern` or something more informative

Reply:
sure